apps/blockchain/lib/mix/tasks/sync_from_file.ex

```diff
@@ -0,0 +1,83 @@
+defmodule Mix.Tasks.SyncFromFile do
+  @shortdoc "Allows users to sync the blockchain from a file exported by parity"
+  @moduledoc """
+  SyncFromFile allows users to sync the blockchain from a file exported by parity.
+
+  You can make an export with the following command:
+
+  `parity export blocks ./export-blocks-mainnet.bin`
+
+  And then run `sync_with_file` to verify the chain:
+
+  `mix run apps/blockchain/scripts/sync_from_file.ex export-blocks-mainnet.bin`
```

Comment:
Should this now be `mix sync_from_file`? Should we change the name to something like `mix sync.from_file` or is that worse?